Cargo.toml

```diff
@@ -6,18 +6,19 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-async-graphql = { version = "3.0.15", features = ["chrono", "uuid"] }
-async-graphql-rocket = "3.0.15"
+async-graphql = { version = "3.0.38", features = ["chrono", "dataloader", "uuid"] }
+async-graphql-rocket = "3.0.38"
```

Comment:
The version `v4` is out! Probably we want to move into that version soon, I tried to land the new version with this PR but theres lots of changes to apply, so I will do in a separate PR.